src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -807,14 +809,46 @@ julia> n_facets(T)
 14
 ```
 """
-archimedean_solid(s::String) = polyhedron(Polymake.polytope.archimedean_solid(s))
+function archimedean_solid(s::String)
+  if s in _archimedean_strings_from_oscar
+    # code used for generation of loaded files can be found at:
+    # https://github.com/dmg-lab/RegularSolidsSrc
+    return getfield(Main, Symbol(s))()
+  end
+  pmp = Polymake.polytope.archimedean_solid(s)
+  return polyhedron(pmp)
+end
+
+@doc raw"""
+      snub_cube()
+
+Construct the Snub Cube, an Archimedean solid.
+- [con24*1](@cite)
+
+See also [`archimedean_solid`](@ref).
+"""
+function snub_cube()
+  return load(joinpath(oscardir, "data", "ArchimedeanSolids", "snub_cube.mrdi"))
```

Comment:
```suggestion
  return archimedian_solid("snub_cube")
```
I would find it much more intuitive the other way around. I.e. `archimedian_solid` checks for `archimedian_strings_from_oscar` and if it is contained directly loads the mrdi file. The named constructors then can directly call this with a string.
This would reduce some duplicate code like the folder name of the solids and the file path construction

Reply:
I think that is probably cleaner than my suggestion.